src/types/params.ts

```diff
@@ -0,0 +1,17 @@
+// Common params across multiple phases, according to
+// https://github.com/babylonlabs-io/pm/blob/main/adr/adr-023-update-btcstaking-params.md
+export interface Params {
+  covenantPks: string[];
```

Comment:
Would be nice to have a type for public keys with and without coordinate with its own validations. Would enhance our type safety. Wonder whether bitcoinjs-lib has anything for that.

Reply:
Agreed, let's track it